spec/factories/menus.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :menu do
+    name "MyString"
+    restaurant
```

Comment:
According to rspec it passes. Not sure what is missing? We currently only have a menu name defined (and an id). And a menu belongs to a restaurant.